setup.cfg

```diff
@@ -1,13 +1,56 @@
-[bdist_wheel]
-universal = 1
-
 [metadata]
+name = pyhf
+version = 0.4.1
+description = (partial) pure Python HistFactory implementation
+long_description = file:README.rst
```

Comment:
```suggestion
long_description = file: README.rst
```

Reply:
@henryiii I'm happy to change this, but just to make sure I'm not missing anything, this is just style/readability right?

Reply:
Just following style here: https://setuptools.readthedocs.io/en/latest/setuptools.html#id47 probably works without a space, never tried. .cfg is not really a well defined language.

Reply:
> probably works without a space

Yeah, it was passing CI but I'm always happy to follow recommended style. Thanks for the review! :)